Fix some typos in documentation (#7156)
onnx/onnx: docs/docsgen/source/intro/python.md

            node.name, node.op_type, node.input, node.output))
```

> The tensor type is a float (= 1). The helper function {func}`onnx.helper.tensor_dtype_to_np_dtype` converts

Reviewer: ```suggestion
The tensor type is an integer value (=1 for `FLOAT`). The helper function {func}`onnx.helper.tensor_dtype_to_np_dtype` converts
```
